hypertools/plot/plot.py

```diff
@@ -59,16 +60,22 @@ def plot(x, fmt=None, marker=None, markers=None, linestyle=None,
         A list of labels for each point. Must be dimensionality of data (x).
         If no label is wanted for a particular point, input None.
 
-    legend : list or bool
-        If set to True, legend is implicitly computed from data. Passing a
-        list will add string labels to the legend (one for each list item).
+    legend : list
+        A list of string labels to be plotted in a legend (one for each list
+        item).
 
     title : str
         A title for the plot
 
     ndims : int
-        An `int` representing the number of dims to plot in. Must be 1,2, or 3.
-        NOTE: Currently only works with static plots.
+        An `int` representing the number of dims to reduce the data x
+        to. If ndims > 3, will plot in 3 dimensions but return the higher
+        dimensional data. Default is None, which will reduce data to 3
```

Comment:
@jeremymanning do we want to set ndims to 3 by default, since it plots and returns 3D data by default?